pages/common/mogrify.md

```diff
@@ -0,0 +1,23 @@
+# Mogrify
+
+> Tool that allows making operations on multiple images ie. resizing, cropping, flipping, adding effects. Replaces the original file.
```

Comment:
I recommended `viz.` because it is often used to cite examples or variants of something. And `ie.` is generally used to explain something in simpler terms.

But it is a minor thing, and if you guys think that `ie.` is well suited here, then I'm okay with it.